tools/build.py

```diff
@@ -343,7 +347,16 @@ def build_iotjs(options):
 
     # --jerry-debugger
     if options.jerry_debugger:
-        cmake_opt.append('-DFEATURE_DEBUGGER=ON')
+        cmake_opt.append("-DFEATURE_DEBUGGER=ON")
```

Comment:
What is the change in this line?

Reply:
Single quote to double quote to be more consistent. It is unrelated, so I can revert is you want.

Reply:
No need. I haven't noticed it.